WooCommerce/Classes/Authentication/Prologue/LoginProloguePageViewController.swift

```diff
@@ -4,9 +4,11 @@ import UIKit
 ///
 final class LoginProloguePageViewController: UIPageViewController {
 
-    private var pages: [UIViewController] = []
+    private let pages: [UIViewController] = {
+        LoginProloguePageType.allCases.map { LoginProloguePageTypeViewController(pageType: $0) }
+    }()
 
-    private var pageControl: UIPageControl!
+    private lazy var pageControl = UIPageControl()
```

Comment:
Can't this be `private let pageControl = UIPageControl()` and just configure it below, and not re-assign it?

Reply:
🤦  Yep, I don't know why I didn't do it that way the first time around! I'll make that change.